molpipeline/mol2mol/scaffolds.py

```diff
@@ -38,6 +46,42 @@ class MakeScaffoldGeneric(_MolToMolPipelineElement):
     Done to make scaffolds less speciffic.
     """
 
+    def __init__(
+        self,
+        generic_atoms: bool = False,
+        generic_bonds: bool = False,
+        name: str = "MakeScaffoldGeneric",
+        n_jobs: int = 1,
+        uuid: Optional[str] = None,
+    ) -> None:
+        """Initialize MakeScaffoldGeneric.
+
+        Note
+        ----
+        Making atoms and or bonds generic allows to check their presence in via SubstructureSearches.
```

Comment:
As already discussed, my suggestion for a better doc-string is: "Making atoms or bonds generic will generate SMARTS strings instead of SMILES strings. This can be useful to search for scaffolds and substructures in data sets. Per default, the scaffold is returned as SMILES string with all atoms set to carbon and all bonds are single bonds."

Reply:
Done. Thx for the text.